app/populate/data/graduation.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function () {
+  return  [{      
+      "curso": "Bacharelado em Ciência e Tecnologia",
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
    "curso": "Bacharelado em Ciência e Tecnologia",
```